src/PiWeb.Volume/PiWeb.Volume.csproj

```diff
@@ -60,7 +57,7 @@
 
   <ItemGroup>
     <None Include="$(SolutionDir)\PiWeb.Volume.Core\lib\*.dll" Pack="true" PackagePath="runtimes\win-x64\native\" Visible="false" />
-    <None Include="$(SolutionDir)\..\bin\$(Configuration)\PiWeb.Volume.Core.dll" Pack="true" PackagePath="runtimes\win-x64\native\" Visible="false" />
+    <None Include="$(SolutionDir)\PiWeb.Volume.Core\bin\$(Configuration)\PiWeb.Volume.Core.dll" Pack="true" PackagePath="runtimes\win-x64\native\" Visible="false" />

```

Comment:
Wird das in beide Ordner ausgegeben oder ist der neue Ordner falsch? Der bin-Ordner liegt eigentlich im Wurzelverzeichnis des Repositories. Ist so ein C++ Ding.